erpnext/regional/india/setup.py

```diff
@@ -27,6 +27,9 @@ def setup_company_independent_fixtures(patch=False):
 	add_print_formats()
 
 def add_hsn_sac_codes():
+	if frappe.flags.in_test and frappe.flags.created_hsn_codes:
```

Comment:
```suggestion
	if frappe.flags.created_hsn_codes:
```

Reply:
I was just being extra-cautious, not touching that flag unless in_test.  😄  But I guess this is fine. It's only set in_test.

Reply:
How about removing the `in_test` on L45 instead?